daemon/graphdriver/driver.go

```diff
@@ -142,10 +142,36 @@ func GetDriver(name, home string, options []string) (Driver, error) {
 func New(root string, options []string) (driver Driver, err error) {
 	for _, name := range []string{os.Getenv("DOCKER_DRIVER"), DefaultDriver} {
 		if name != "" {
+			logrus.Infof("[graphdriver] trying provided driver %q", name) // so the logs show specified driver
 			return GetDriver(name, root, options)
 		}
 	}
 
+	// Guess for prior driver
```

Comment:
though, the specified (e.g. `-s <DRIVER>` and `DOCKER_DRIVER=<DRIVER>`) has already been resolved above this though.

Are you recommending to output to a config file? I don't think that is needed. Also, I've added Info for recording this data, for easier troubleshooting. The only thing that would make this "scary" is if someone is using various drivers in the same directory, and then _hoping_ that by not specifying a driver that docker would make the _right_ choice (for some definition of _right_)